### ISSUE TITLE
Alt Text for links and images of items in the carousel

### ISSUE THREAD

**Issue by bmaass97** [omeka-s-modules/ItemCarouselBlock#27]
For accessibility requirements related to ADA Title II and WCAG 2.1 Level AA, all images and links should have alt text.

In the carousel, the linked images aren't including alt text, even if the “Media alt text property” in Global Settings is set. 

Other images (outside of carousels) throughout our pages do show alt text and appear to fall back to the file name if other alt text isn't present. 

This forum message explains with links to live example: 
    https://forum.omeka.org/t/alt-text-for-item-carousel-block-module/29108

**Comment by bmaass97** [comment]
Sorry...I confused the **Item** metadata and the **Media** metadata (which we hadn't been adding because we just attach the media when we create the Item record).
